# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/browser.toml
# Commit: 98848e63dfbf
# Full Hash: 98848e63dfbf5702726d3e07fb92855263312fed
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r b668d0c685e8 -r 98848e63dfbf netwerk/cookie/test/browser/browser.toml
--- a/netwerk/cookie/test/browser/browser.toml	Wed Jan 08 19:25:59 2025 +0000
+++ b/netwerk/cookie/test/browser/browser.toml	Wed Jan 08 19:26:00 2025 +0000
@@ -6,6 +6,9 @@
   "head.js"
 ]
 
+["browser_3pcb_expection.js"]
+support-files = ["setFetchCookie.sjs"]
+
 ["browser_broadcastChannel.js"]
 
 ["browser_chips_partition_cap_child.js"]
@@ -13,6 +16,9 @@
 ["browser_cookie_chips.js"]
 support-files = ["chips.sjs"]
 
+["browser_cookie_empty_name_value.js"]
+support-files = ["cookie_empty_name_value.sjs"]
+
 ["browser_cookie_insecure_overwrites_secure.js"]
 
 ["browser_cookie_purge_sync.js"]
@@ -34,9 +40,6 @@
 ["browser_oversize.js"]
 support-files = ["oversize.sjs"]
 
-["browser_cookie_empty_name_value.js"]
-support-files = ["cookie_empty_name_value.sjs"]
-
 ["browser_partitionedConsole.js"]
 support-files = ["partitioned.sjs"]
 